modules/scripts/startup-script/templates/startup-script-custom.tpl

```diff
@@ -6,7 +6,7 @@ stdlib::run_playbook() {
     "Please install ansible before running ansible-local runners."
     exit 1
   fi
-  /usr/bin/ansible-playbook --connection=local --inventory=localhost, --limit localhost $1
+  /usr/bin/ansible-playbook --connection=local --inventory=localhost, --limit localhost $1 $2
```

Comment:
I did some testing, and I believe this will allow further `--limit ...` flags, which would allow this to run non-locally. Can a validation be added to ensure `--connection=.*`, `--inventory=.*` or `--limit .*` are not included in args for ansible?

I'm also open to allowing that since it's already possible to do that via a shell script as is.

Reply:
How do you feel about addressing this in README for now? I see a note about args only applying to `shell` that needs to be updated anyways.

Reply:
Sure, that seems reasonable.

Reply:
(I will squash these commits upon merge, FYI)